common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -587,13 +733,32 @@ def test_can_show_answer(self):
         self.problem_page.wait_for_show_answer_notification()
 
 
+class CheckboxProblemTypeTestNonRandomized(CheckboxProblemTypeBase, NonRandomizedProblemTypeTestMixin):
+    """
+    Tests for the non-randomized checkbox problem
+    """
+
+    def get_problem(self):
+        """
+        Creates a {problem_type} problem
+        """
+        # Generate the problem XML using capa.tests.response_xml_factory
```

Comment:
I can see this multiple times. Do we really need this string? I think it is obvious what we are returning by looking at code.